Makefile

```diff
@@ -159,7 +181,7 @@ bundle-validate: container-cmd operator-sdk
 .PHONY: build
 build: generate fmt vet ## Build manager binary.
 	go build -o bin/manager cmd/manager/main.go
-	go build -o bin/sidecar sidecar/main.go
+	go build -ldflags '$(LDFLAGS)' -o bin/sidecar sidecar/main.go
```

Comment:
can you add this to the manager as well?

Reply:
added to manager as well.